src/main/scala/intellij/haskell/external/repl/ProjectStackReplProcess.scala

```diff
@@ -47,20 +48,24 @@ private[repl] class ProjectStackReplProcess(project: Project) extends StackReplP
   }
 
   private def load(psiFile: PsiFile, moduleName: Option[String]): Option[(StackReplOutput, Boolean)] = {
-    val filePath = getFilePath(psiFile)
-    val moduleNameAction = ApplicationManager.getApplication.runReadAction(new Computable[Option[String]] {
-      override def compute(): Option[String] = {
-        moduleName.orElse(HaskellPsiUtil.findModuleName(psiFile))
+    if (!HaskellConsoleView.isConsole(psiFile)) {
```

Comment:
@rikvdkleij Actually, the current approach is the best one I think, If I do the console file judgment here in [HaskellAnnotator](https://github.com/rikvdkleij/intellij-haskell/blob/master/src/main/scala/intellij/haskell/annotator/HaskellAnnotator.scala#L78), it will expose the psi file to `:load`, `:type-at`, `:loc-at`, but the changes here can avoid all these things.

![image](https://cloud.githubusercontent.com/assets/6234553/23272665/20c65efa-fa37-11e6-8e62-6ca4b6448171.png)
![image](https://cloud.githubusercontent.com/assets/6234553/23272671/2720a30a-fa37-11e6-993b-a4f2964d6a46.png)